wd_cipher.c

```diff
@@ -415,6 +418,9 @@ int wd_do_cipher_sync(handle_t h_sess, struct wd_cipher_req *req)
 	} while (ret < 0);
 	pthread_spin_unlock(&ctx->lock);
 
+	if (wd_cipher_env_config.enable_internal_poll)
+		wd_add_task_to_async_queue(&wd_cipher_env_config, idx);
+
```

Comment:
I don't understand why add it into async queue while you're running in sync mode. Do you write it in the wrong place?

Reply:
It is a mistake when sync code,I have changed it